packages/icons/src/serverless-aws-lambda-package-inverse.svg

```diff
@@ -0,0 +1 @@
+<svg width="33" height="32" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M5.727 5.5a.5.5 0 0 1 .5-.5h7.875a.5.5 0 0 1 .416.223l2.476 3.715h10.233a.5.5 0 0 1 .5.5v10.141a.5.5 0 1 1-1 0V9.938h-7.212v1.701h1.157a.5.5 0 0 1 0 1h-1.157v1.275h1.157a.5.5 0 1 1 0 1h-1.157v1.099h1.157a.5.5 0 0 1 0 1h-1.157v1.276a.5.5 0 1 1-1 0v-1.276h-1.158a.5.5 0 1 1 0-1h1.158v-1.1h-1.158a.5.5 0 1 1 0-1h1.158V12.64h-1.158a.5.5 0 1 1 0-1h1.158V9.938h-1.788a.5.5 0 0 1-.416-.223L13.834 6H6.727v20h13.392a.5.5 0 0 1 0 1H6.227a.5.5 0 0 1-.5-.5v-21z" fill="#fff"/><path d="m23.773 32 6.01-5.443-.447-.28-5.094 4.61v-1.09l5.658-5.119v1.099l-.564.5.447.28.602-.535v-2.123l-1.318.182 1.181-1.978v-1.866L29.986 20l-6.918 5.96v1.46l2.22-.375-1.516 2.563V32zm5.991-10.81v.768l-1.268 2.201-.645.09 1.913-3.06zm-6.23 5.646v-.788l2.7-.368-.567.818-2.134.338z" fill="#fff"/></svg>
```

Comment:
Let's remove width and height and use viewBox instead

Reply:
Done.